### ISSUE TITLE
Different zones

### ISSUE THREAD

**Issue by d-starilov** [tikv/tikv#15352]
I have 3 data centers (A, B and C) and one cloud for storing the data. I've created 3 tikv-servers with PD on it (1/per data center), connected it all, mounted cloud to servers using juicefs, but noticed that servers frome dc A connects to PD-A, but goes for files via main PD server (in dc B) which slows whole work because of waiting for files to be downloaded. In this case I have a question, is there any possibility to set PD server to work only with it's zone? Like if server from zone C connects to PDC, PDC sends him to work only with tikva-server in zone C

**Comment by Smityz** [comment]
I think the answer is no, only one PD server can be the leader. Are there any log or monitoring indicators indicating that the slow speed is caused by slow file download on PD？

**Comment by Fa1c0nE** [comment]
No, I do not have any logs but I know for sure the problem is about speed between zones. So I was looking for a way to set different PD servers for each zone, kind of duplicating. Each server connects to a cluster in it's own zone and each cluster connected with others
